_posts/2018-11-16-stairway-to-health.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+authors: [gert-jan_schokkaert]
+title: "Visualising IoT data with tableau"
+image: /img/2018-11-16-stairway-to-health/stairway-to-health.png
+tags: [Development,Translations,Angular,i18n,Crowdin]
+category: Development
+comments: true
+---
+
+# VisionWorks meets JWorks: StairwayToHealth
+We from [VisionWorks][] were asked to rebuild the visualisation dashboard [JWorks][] used in the [application][] they built as a result of the internal StairwayToHealth project (you can find more information about that project [here][]). We decided to use [Tableau][], a popular BI Visualisation tool we largely use at our clients.
+We developed the dashboard working around some key questions while keeping the appearance of the dashboard in line with the dashboard JWorks developed.
+
+In the following section I will explain how the dashboard is currently set up and how to use it properly.
```

Comment:
Keep the subject in line with previous section "We from VisionWorks" --> Following section "We will explain"